src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/components/RepeatingGroupsAddButton.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-undef */
```

Comment:
Could we rename file to `RepeatingGroupAddButton.tsx` to be consistent with `RepeatingGroupTable.tsx`? 🤓

Reply:
Yes!